### PR TITLE
Add ARB and OP

### PR DESCRIPTION
**Pull request by akuter-bc**

### CODE REVIEW

**Review by jjambazov-bc — APPROVED** [review]

File: config-staging.json [hunk 1]
```diff
@@ -89,6 +89,46 @@
...
+      "explorerUrl": "https://optimistic.etherscan.io/tx",
+      "feeCurrencies": [
+        "native"
+      ],
+      "identifiers": {
+        "chainId": 10
+      },
+      "memos": false,
+      "name": "Optimism",
+      "nativeAsset": "OP",
```
Comment (jhuntbach-bc) [inline config-staging.json:123]: Isn't the native asset for Optimism actually `ETH.OP`?

File: config-staging.json [hunk 2]
```diff
@@ -89,6 +89,46 @@
...
+      "feeCurrencies": [
+        "native"
+      ],
+      "identifiers": {
+        "chainId": 42161
+      },
+      "memos": false,
+      "name": "Arbitrum",
+      "nativeAsset": "ARB",
+      "networkTicker": "ARB",
```
Comment (jhuntbach-bc) [inline config-staging.json:104]:
```suggestion
      "networkTicker": "ARBETH",
```
I think this needs to match the coin-definitions entry: https://github.com/blockchain/coin-definitions/blob/master/scripts/statics.py#L69

**Review by jhuntbach-bc — CHANGES_REQUESTED** [review]
Update native assets and arbitrum ticker

File: config-staging.json (hunk 1 above)
Comment (mbratfisch-bc) [reply config-staging.json:123]: That sounds correct. OP is a governance token as far as I remember.

**Review by jjambazov-bc — APPROVED** [review]

File: config-staging.json [hunk 3]
```diff
@@ -89,6 +89,46 @@
...
+      "explorerUrl": "https://arbiscan.io/tx",
+      "feeCurrencies": [
+        "native"
+      ],
+      "identifiers": {
+        "chainId": 42161
+      },
+      "memos": false,
+      "name": "Arbitrum",
+      "nativeAsset": "ETH.ARB",
```
Comment (jhuntbach-bc) [inline config-staging.json:103]: I think this should be `ARBETH`
https://github.com/blockchain/coin-definitions/blob/master/coins.json#L43

File: config-staging.json [hunk 4]
```diff
@@ -89,6 +89,46 @@
...
+      "explorerUrl": "https://optimistic.etherscan.io/tx",
+      "feeCurrencies": [
+        "native"
+      ],
+      "identifiers": {
+        "chainId": 10
+      },
+      "memos": false,
+      "name": "Optimism",
+      "nativeAsset": "ETH.OP",
```
Comment (jhuntbach-bc) [inline config-staging.json:123]: This doesn't exist in coin-definitions yet

**Review by jjambazov-bc — APPROVED** [review]

**Review by jhuntbach-bc — DISMISSED** [review]

**Review by jhuntbach-bc — DISMISSED** [review]

**Review by jhuntbach-bc — APPROVED** [review]

**Pull request merged**